server/netty-server/src/main/scala/sttp/tapir/server/netty/internal/ReactiveWebSocketHandler.scala

```diff
@@ -0,0 +1,182 @@
+package sttp.tapir.server.netty.internal
+
+import io.netty.buffer.Unpooled
+import io.netty.channel.group.ChannelGroup
+import io.netty.channel.{ChannelHandlerContext, ChannelInboundHandlerAdapter}
+import io.netty.handler.codec.http._
+import io.netty.handler.codec.http.websocketx.WebSocketServerHandshakerFactory
+import io.netty.handler.timeout.ReadTimeoutHandler
+import io.netty.util.ReferenceCountUtil
+import org.playframework.netty.http.DefaultWebSocketHttpResponse
+import org.slf4j.LoggerFactory
+import sttp.monad.MonadError
+import sttp.monad.syntax._
+import sttp.tapir.server.model.ServerResponse
+import sttp.tapir.server.netty.NettyResponseContent.ReactiveWebSocketProcessorNettyResponseContent
+import sttp.tapir.server.netty.{NettyResponse, NettyResponseContent, NettyServerRequest, Route}
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.control.NonFatal
+import scala.util.{Failure, Success}
+
+/** Handles a WS handshake and initiates the communication by calling Tapir interpreter to get a Pipe, then sends that Pipe to the rest of
+  * the processing pipeline and removes itself from the pipeline.
+  */
+class ReactiveWebSocketHandler[F[_]](
```

Comment:
is this based on some cats-effects class? if so we should add a backlink

Reply:
No, I wrote it from scratch, is that what you mean?

Reply:
ah ok :) the comment "By using the Netty ..." looked formatted differently than our other comments so I thought that's taken from somewhere - sorry :)